src/ConnectCore.js

```diff
@@ -135,7 +140,8 @@ class ConnectCore {
    */
   requestCredentials (request = {}, uriHandler) {
     const topic = this.topicFactory('access_token')
-    if (this.accountType) request.accountType = this.accountType
+    request.accountType = this.accountType || request.accountType || 'none'
```

Comment:
This should swap this.accountType and request.accountType. Having it in the request should override the one in the overall settings.

Reply:
Ah just kidding, I see now !

Reply:
Actually thinking about it again, this does mean that as written, `subProvider.getAddress` will always use `accountType: 'keypair'`.  Is there a situation where we would want it to use something else ?